### PR TITLE
Add stream uptime to stream information 

### LINKED ISSUE #491: Enhancement: Show stream uptime in stream information
Show the stream uptime in the stream information below the chat input.

I have added it in a fork [here](https://github.com/CrippledByte/DankChat/commit/77bcf68a30e2965b9ab55d1de5de8b133e63a564) and [updated it](https://github.com/CrippledByte/DankChat/commit/4f9d99d28cd90a51f5d1cb1205de06429871b90b) with [code from CommandRepository.kt](https://github.com/flex3r/DankChat/blob/c41a9bb418d37978af74427b687f1fe178c00133/app/src/main/kotlin/com/flxrs/dankchat/data/repo/command/CommandRepository.kt#L243). I have just hard-coded the string for now, and I am not sure how to properly handle possible errors, but I thought it may be useful so wanted to share it here.

It looks like this:
drawing
drawing
drawing

### PR DESCRIPTION
**Pull request by CrippledByte**
This shows the stream uptime in the stream information below the chat input (issue #491). I updated the string resource to include the uptime and moved code that calculates the uptime to DateTimeUtils.kt.
Closes #491

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -410,8 +410,8 @@
     <string name="confirm_login_cancel_message">Are you sure you want to cancel the login process?</string>
     <string name="confirm_login_cancel_positive_button">Cancel login</string>
     <plurals name="viewers">
-        <item quantity="one">Live with %d viewer</item>
-        <item quantity="other">Live with %d viewers</item>
+        <item quantity="one">Live with %d viewer for %s</item>
```
Comment (flex3r) [inline app/src/main/res/values/strings.xml:413]: I think we unfortunately have to introduce a completly new string resource in order to not break the existing translations that don't have the additional argument

**Review by flex3r — CHANGES_REQUESTED** [review]
Looks good, we just need to make sure to not break the existing translations for the old string variant. No need for any exception handling for the duration parsing, imo 👍

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (CrippledByte) [reply app/src/main/res/values/strings.xml:413]: Would a new string resource replace the current one?

**Review by flex3r — APPROVED** [review]
Nice, this is good to go 👍 
I'm going to wait with merging for a bit until I have the current beta promoted to production.
Thanks for the help and feel free to message me on Twitch #flex3rs if you want a contributor badge!

**Comment by CrippledByte** [comment]
Nice, thanks for your help 👍

**Pull request merged**